app/views/groups/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1 class="text-primary">Create Category</h1>
+
+<%= render "form", group: @group %>
+
+<br>
+
+<div>
+  <%= link_to "Back to categories", groups_path, class: "btn btn-primary" %>
+</div>
```

Comment:
- [ ] When you are creating the category, you don't have to include the amount, you have to add the expenses on the `new transaction`  

![image](https://user-images.githubusercontent.com/55635977/154804340-a3d01433-c2b4-4639-a8df-1314a66fc66d.png)

Reply:
Thank you for the sugestion. i have implemented this change